src/classes/ACCT_CannotDelete_TEST.cls

```diff
@@ -961,7 +1025,10 @@ public with sharing class ACCT_CannotDelete_TEST {
         insert term; 
 
         Time_Block__c timeBlock = new Time_Block__c (Name = 'TimeB', Educational_Institution__c = lastNameByContact.get('Willy').AccountId);
-        insert timeBlock; 
+        insert timeBlock;
+
+        Education_History__c educationHistory = UTIL_UnitTestData_TEST.getEduHistory(lastNameByContact.get('Willy').AccountId, null);
```

Comment:
Note that for the large-scale integration test, one record is fine, unless we want to build a engine to provide multiple records at each phase.